Python/Product/PythonTools/PythonToolsService.cs

```diff
@@ -670,6 +670,18 @@ public sealed class PythonToolsService : IDisposable {
                 pathVar = "PYTHONPATH";
             }
             baseEnv[pathVar] = string.Empty;
+
+            // TODO: We could introduce a cache so that we don't activate the
+            // environment + capture the env variables every time. Not doing this
+            // right now to minimize risk/complexity so close to release.
```

Comment:
Did you create an issue for it?

Reply:
Done, thanks
https://github.com/Microsoft/PTVS/issues/5124